test/cache-tests.js

```diff
@@ -71,7 +71,7 @@ function basicTests(factory) {
 
     it('should store and retrieve binary buffers', () => {
         const cache = factory();
-        const buffer = Buffer.alloc(2 * 1024 * 1024);
+        const buffer = Buffer.alloc(2 * 1024);
```

Comment:
These changes just sped things up?

Reply:
Yep - pretty much just to make the red duration nags go away on these tests. It only saves a few 10s of milliseconds in the end